### PR TITLE
[WIP] restructure Quick Start

### PR DESCRIPTION
**Pull request by StackScribe**
Signed-off-by: Meg McRoberts <meg.mcroberts@dynatrace.com>

This is a PoC draft to share with @Dynatrace-Adam-Gardner and anyone else who is interested.

We are restructuring the Quick Start to only use a containerized installation of Keptn -- either the Helm chart or a Docker container.  We tell them that they can set up their own k8s cluster and install Keptn on it but refer them to the installation page for information.  That page will be updated to discuss creating a local Keptn installation for study and demonstration purposes.

We then expand the discussion of what the two exercises do.  The specifics of that content need to be fixed but the current content shows the flavor.

I have not yet modified the content that follows the exercises although I expect that most of it will be moved out of the Quick Start and the Quick Start will point them to information about how to create a project.

### CODE REVIEW

File: content/docs/quickstart/_index.md [hunk 1]
```diff
@@ -1,22 +1,35 @@
...
-and run some exercises to demonstrate Keptn functionality.
+This Quick Start guide enables you to quickly access a Keptn environment locally
+and run some simple exercises that demonstrate Keptn functionality.
+The steps are:
 
-You can install and run Keptn on virtually any Kubernetes cluster.
-See Install CLI and Keptn for detailed instructions
-about creating a Keptn cluster locally or in the cloud.
+* Download a container (either Docker or Kubernetes)
+that providea a small but complete Kubernetes cluster with Keptn installed.
```
Comment (agardnerIT) [inline content/docs/quickstart/_index.md:15]: small typo: `providea` > `provides`

File: content/docs/quickstart/_index.md [hunk 2]
```diff
@@ -1,22 +1,35 @@
...
+* Run a simple exercise that runs a Quality Gate
+against a multi-stage delivery operation.
+* Run another exercise that shows how the Keptn auto-remediation feature
+can automatically fix problems that the quality gates report.
 
-## Helm
+## Install Keptn in a container
+
+We provide a Helm chart and a Docker container that you can install locally
```
Comment (agardnerIT) [inline content/docs/quickstart/_index.md:23]: We provide a Docker container that you can install locally. Alternatively, we provide a Helm chart that can be used to install on your own cluster...

**Comment by agardnerIT** [comment]
Also touches on: `https://github.com/keptn/keptn/issues/7176`

File: content/docs/quickstart/_index.md (hunk 1 above)
Comment (StackScribe) [reply content/docs/quickstart/_index.md:15]: Fixed

File: content/docs/quickstart/_index.md (hunk 2 above)
Comment (StackScribe) [reply content/docs/quickstart/_index.md:23]: Applied

**Comment by StackScribe** [comment]
> Also touches on: [keptn/keptn#7176](https://github.com/keptn/keptn/issues/7176)
Yup, and also the work being done to provide a katacoda environment that could be used...
Do those need to be mentioned in this document?  Perhaps it would be nice to estimate
how long it takes to spin up either the Docker container or the helm chart?

**Comment by agardnerIT** [comment]
> (katacoda): Do those need to be mentioned in this document?
Yes potentially, but we can add that if / when katacoda gets into a stable state.

> Perhaps it would be nice to estimate how long it takes to spin up either the Docker container or the helm chart?
Short answer: It's hard to say. `helm deploy` will just ask the kubernetes cluster to download and deploy whatever images are referenced in the chart. So really the question is then down to the speed of your internet connection and cluster as to how fast the images are retrieved from the image registry and how much RAM your cluster has as to how fast those containers start.

**Comment by StackScribe** [comment]
So a comment like "This may take 5-10 minutes, depending on the speed of your Internet and cluster" or something.  As opposed to taking an hour or something.  What I want to know is:
* Should I sit here and wait for it?
* Should I go get a beverage or snack and come back?
* Should I go to lunch while this initializes?
* Should I go to bed and look forward to having an installation when I wake up?
Those are my units of time ;-). And, while faster is better, I can live with any of them if I know what to expect.

**Comment by StackScribe** [comment]
This work is superseded by the initiative to use KataCoda or Kilrcoda for all the tutorials (https://github.com/keptn/keptn.github.io/milestone/2) so I am closing this PR.

**Pull request closed without merging**